src/firebase-config.js

```diff
@@ -0,0 +1,15 @@
+import { initializeApp } from "firebase/app";
+import {getFirestore} from '@firebase/firestore'
+const firebaseConfig = {
+    apiKey: "AIzaSyBXzt7OQ5sw13u77Lj7C3elMeDyWrgxxYg",
+    authDomain: "vaspacx-quiz-react.firebaseapp.com",
+    projectId: "vaspacx-quiz-react",
+    storageBucket: "vaspacx-quiz-react.appspot.com",
+    messagingSenderId: "528289058321",
+    appId: "1:528289058321:web:e7da60affe61e846adabb0",
+    measurementId: "G-6MTMXQLZ8M"
```

Comment:
This private information can be stored in an .env file which will not be pushed to Github and shall be added to the platform where you are deploying the site
An example of how .env looks like can be taken from this project:

```
REACT_APP_API_KEY = AIzaSyBXzt7OQ5sw13u77Lj7C3elMeDyWrgxxYg
REACT_APP_AUTH_DOMAIN = vaspacx-quiz-react.firebaseapp.com
REACT_APP_PROJECT_ID = vaspacx-quiz-react
REACT_APP_STORAGE_BUCKET = vaspacx-quiz-react.appspot.com
REACT_APP_MESSAGING_SENDER_ID = 528289058321
REACT_APP_APP_ID = 1:528289058321:web:e7da60affe61e846adabb0
REACT_APP_MEASUREMENT_ID = G-6MTMXQLZ8M
```